Makefile

```diff
@@ -0,0 +1,14 @@
+lint:
+	# black should be last in the list, as it lint the code. Tests can fail if order will be different
+	flake8 && isort . && black .
```

Comment:
Do we want a workflow that checks these?

Reply:
This is just for developers, and we are testing only black. I would leave it here

Reply:
I'm not sure I like this makefile here, just since this isn't something we're doing in other projects. But if you want to keep it, fine.